pkg/parser/parser.go

```diff
@@ -130,19 +130,16 @@ func (p *PackageParser) Parse(_ context.Context, reader io.ReadCloser) (*Package
 	return pkg, nil
 }
 
-// cleanYAML cleans up YAML by removing empty and commented out lines which
+// cleanYAML cleans up YAML only drops fully commented inputs which would
```

Comment:
This is reading a little off now - is it correct to say the following? I'm trying to be more complete about what this function is doing and returning - feel free to disregard the completely aspect if its wrong or you don't think its needed, but the sentence as is is missing some flow 🤓 

```suggestion
// cleanYAML cleans up the input YAML by removing all leading and trailing white
// space from each line. If the input yaml is completely commented, then nil
// will be returned to avoid issues with decoding.
```